providers/implementations/signature/rsa_sig.c

```diff
@@ -327,14 +327,21 @@ static int rsa_setup_md(PROV_RSA_CTX *ctx, const char *mdname,
                            "digest=%s", mdname);
             goto err;
         }
+        /* XOF digests are not allowed except for RSA PSS */
+        if (ctx->pad_mode != RSA_PKCS1_PSS_PADDING
```

Comment:
Unless we support this it will probably just fall over later??

Reply:
It will, but it's on NIST's permitted list.  Thus we should make a token effort to support it.

Reply:
I think we should support only what we allow? Not all algorithms need to be supported.

Reply:
Updated.